### PR TITLE
ui: Enable theming

### PR DESCRIPTION
**Pull request by johncowen**
This PR enables theming for the UI.

Please note: This is not yet a user facing feature and there is no user facing switch to be able to switch theme. Although this PR includes a temporary dev bookmarklet, which enables us to switch/preview theming and iterate on the very basic dark/light theming we've had in the UI for a little while.

All in all this will eventually make things look _epic_! (@lkysow 😸 )

Screenshot 2021-11-18 at 11 09 42

The dev bookmarklet is just temporary to make it easy for me to switch (I have a few related features to build that should be built with theming in mind), but until we have an actual using facing widget to switch if anyone else wanted to see/try they could just add the `consul:theme` localStorage value also (please note, this is a work in progress and not currently supported and a gradual iterative piece of work, YMMV etc etc).

Screenshot 2022-01-20 at 10 13 14

cc @david-yu 😆

### CODE REVIEW

File: ui/packages/consul-ui/app/utils/get-environment.js
Comment (johncowen) [inline ui/packages/consul-ui/app/utils/get-environment.js:97]: :prettier-da:

**Review by randallmorey — APPROVED** [review]
❤️ 🕶️  So pretty!  It could use a tad more color.  For reference:  https://en.wikipedia.org/wiki/LCARS

**Comment by johncowen** [comment]
> So pretty! It could use a tad more color.

😆 oh yes definitely, we just reversed all our colors while back, not great but it gets us to a good starting point to iterate and refine 🎉 

ty! (will wait for the icon one to come down on here before I merge this one)

**Comment by amyrlam** [comment]
Aside to the color comment: If you make any color tweaks, and want to trend toward the design system, we've started documenting some of them: https://design-system-components-git-add-tokens-documentation-hashicorp.vercel.app/foundations/tokens (from https://github.com/hashicorp/design-system-components/pull/141, to be merged in soon)

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/561743.

**Pull request merged**
